crates/corro-types/src/change.rs

```diff
@@ -232,12 +232,29 @@ pub fn insert_local_changes(
         version: Some(version),
     })?;
 
+    let mut cleared_ts: Option<Timestamp> = None;
     for (actor_id, versions_set) in overwritten {
+        if actor_id != agent.actor_id() {
+            warn!("clearing and setting timestamp for empties from a different node");
+        }
         for versions in versions_set {
-            store_empty_changeset(tx, actor_id, versions)?;
+            let ts = Timestamp::from(agent.clock().new_timestamp());
+            let inserted = store_empty_changeset(tx, actor_id, versions, ts)?;
+            if inserted > 0 {
+                cleared_ts = Some(ts)
+            }
         }
     }
 
+    if let Some(ts) = cleared_ts {
+        snap.update_cleared_ts(tx, ts)
```

Comment:
just to be sure: this is updating the sync table, but for the current node. so I suppose it's useful if we do a snapshot so the restoring node is already up to date for this node.

Reply:
yes, it updating the sync for the current node.